import-export-cli/integration/app_test.go

```diff
@@ -480,3 +480,27 @@ func TestDeleteApp(t *testing.T) {
 		})
 	}
 }
+
+// Export an application with space in application name  and import it to another  to check whether the url
+// encoding is working properly
+func TestExportImportOwnAppWithSpaceInAppName(t *testing.T) {
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+			prod := GetProdClient()
+
+			app := testutils.AddAppWithSpaceInAppName(t, dev, user.CtlUser.Username, user.CtlUser.Password)
```

Comment:
Shall we use the Subscriber user here instead of the Ctl user? (Because when the looping happens same thing will be happening twice again and again)